_includes/analytics.html

```diff
@@ -1,9 +0,0 @@
-<!-- Google tag (gtag.js) -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-YFX0BFPME2"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'G-YFX0BFPME2');
```

Comment:
Please add the Google Analytics tag to the site.  For details see [here](https://github.com/adityatelange/hugo-PaperMod/discussions/248#discussioncomment-11781262).